packages/p2p-v2/src/pages/orders/components/ChatFooter/ChatFooter.tsx

```diff
@@ -1,14 +1,18 @@
-import React, { ChangeEvent, useState } from 'react';
+import React, { ChangeEvent, KeyboardEvent, useRef, useState } from 'react';
 import { Input, Text, useDevice } from '@deriv-com/ui';
 import ChatFooterIcon from '../ChatFooterIcon/ChatFooterIcon';
 import './ChatFooter.scss';
 
 type TChatFooterProps = {
     isClosed: boolean;
+    sendFile: (file: File) => void;
+    sendMessage: (message: string) => void;
 };
-const ChatFooter = ({ isClosed }: TChatFooterProps) => {
+const ChatFooter = ({ isClosed, sendFile, sendMessage }: TChatFooterProps) => {
     const { isMobile } = useDevice();
     const [value, setValue] = useState('');
+    const fileInputRef = useRef<HTMLInputElement>(null) as React.MutableRefObject<HTMLInputElement | null>;
+    const textInputRef = useRef<HTMLInputElement>(null) as React.MutableRefObject<HTMLInputElement | null>;
```

Comment:
Does this work @nada-deriv? 🤔 
```suggestion
    const fileInputRef = useRef<HTMLInputElement | null>(null);
    const textInputRef = useRef<HTMLInputElement | null>(null);
```